### DISCUSSION TITLE
No image c64

### DISCUSSION THREAD

**Discussion by FireflySE** [frntc/Sidekick64#100] [Q&A]
Hi 

I have assembled 2 pcb but I get no image on c64. Raspberry zero 2w have image with the logo so i guess boot is not an issue. led lights are alternating slowly and reset light is a little more dim then the other. I rechecked pcb for bad connections but looks ok. 🤔 Any suggestions how to proceed?

**Comment by RaymondDay61** [comment]
Same here can down load the .zip file is do you drag and drop them on a fat32 format SD card? Or just install a RP image on it and then put the files were on that? In /home/user don't see any thing that says this. There should be a Sidekick64.img like a DD image to start on a SD card. Download the "Sidekick64-0.51d" The files in there do you just put in a fat32 SD card root folder or what?

**Comment by RaymondDay61** [comment]
I did do that formatted a 64 GB SD card to fat and drag and drop the files on there but only the rest button works and no LCD little display. Is there some DD image can download some were?

**Reply by FireflySE** [reply]
The zip file is a bootable image just extract to your sd card, but remember that you have to edit the config file and choose display. I used vscode for that you also have instructions in the config file.

**Comment by RaymondDay61** [comment]
I used Raspberry Pi imager V1.8.5 and said "Sidekick64-0.51d.zip has been written to Generic STORAGE DEVICE USB Device" So I guess that is the way to do it. I did not know a .zip file could be a image like that.

Put that SD card in my C=64 with the Sidekick64 and still don't work.

**Comment by nosyn00b** [comment]
I had the same "blank screen"  experience due to defective 257s. I solved this (starting from _frntc_ suggestions) by measuring TTL levels on the expansion port of the C64 and replacing the 257s connected to lines that wrongly had not at least 4Vpkpk signals where expected  (for me it happened on IO1, IO2 and also A0-A3 lines on different PCBs).  EXROM and GAME levels should also be GND or +5V correctly (both low to GND when on the main SK64 Menu if I remember correctly). I also suggest carefully checking with a multimeter the continuity from each of the 257s pins to the Expansion port or GPIO port etc.
